centaur/src/main/resources/standardTestCases/cacheBetweenWfNoCost.test

```diff
@@ -0,0 +1,17 @@
+name: cacheBetweenWFNoCost
+testFormat: runtwiceexpectingcallcachingnocost
+backends: [Papi, GCPBATCH]
```

Comment:
Tags are a good way to include/exclude a cohort of tests from a particular suite.

[Here](https://github.com/broadinstitute/cromwell/blob/408951fa2d5baf10b18ba1a104c0219262d83c80/src/ci/bin/testCentaurHoricromtalPapiV2beta.sh#L23-L29) is where to exclude.

```suggestion
backends: [Papi, GCPBATCH]
tags: [cost]
```